src/main/java/at/hannibal2/skyhanni/features/misc/FrogMaskFeatures.kt

```diff
@@ -79,5 +106,19 @@ object FrogMaskDisplay {
         )
     }
 
-    private fun isEnabled() = IslandType.THE_PARK.isInIsland() && config.frogMaskDisplay
+    @HandleEvent
+    fun onConfigFix(event: ConfigUpdaterMigrator.ConfigFixEvent) {
+        event.move(74, "misc.frogMaskDisplay", "misc.frogMaskFeatures.frogMaskDisplay")
+        event.move(74, "misc.frogMaskDisplayPosition", "misc.frogMaskFeatures.frogMaskDisplayPosition")
+    }
+
+    fun isForaging(): Boolean {
+        if (SkillApi.activeSkill != SkillType.FORAGING) return false
+        val info = SkillApi.skillXPInfoMap[SkillType.FORAGING] ?: return false
+        return info.lastUpdate.passedSince() < 10.seconds
+    }
+
+    fun WarningType.isSelected() = config.warning.warningType.contains(this)
+
+    private fun isEnabled() = IslandType.THE_PARK.isInIsland() && (config.display || config.warning.enabled)
```

Comment:
Remove `IslandType.THE_PARK.isInIsland()` from this fun, and add it as an `onlyOnIsland` to the event receiver